### PR TITLE
Capacity parser - CA-ON

### PR DESCRIPTION
**Pull request by mathilde-daugy**
## Issue

Capacity parser for CA-ON

## Description

Capacity parser using the same logic as others (see parser for [GB](https://github.com/electricitymaps/electricitymaps-contrib/pull/6092))

### Preview

image

### Double check

- [ ] I have tested my parser changes locally with `poetry run test_parser "zone_key"`
- [ ] I have run `pnpx prettier --write .` and `poetry run format` to format my changes.

### CODE REVIEW

File: electricitymap/contrib/capacity_parsers/CA_ON.py [hunk 1]
```diff
@@ -0,0 +1,65 @@
...
+        return df
+    else:
+        raise ValueError(
+            f"Failed to fetch capacity data for IESO from url: {target_datetime.date()}"
+        )
+
+
+def fetch_production_capacity(
+    zone_key: ZoneKey, target_datetime: datetime, session: Session
+) -> Dict:
```
Comment (VIKTORVAV99) [inline electricitymap/contrib/capacity_parsers/CA_ON.py:43]:
```suggestion
) -> dict:
```

File: electricitymap/contrib/capacity_parsers/CA_ON.py [hunk 2]
```diff
@@ -0,0 +1,65 @@
...
+            "source": "ieso.ca",
+        }
+    logger.info(
+        f"Fetched capacity for {zone_key} on {target_datetime.date()}: \n{capacity}"
+    )
+    return capacity
+
+
+if __name__ == "__main__":
+    fetch_production_capacity("CA_ON", datetime(2023, 3, 1), Session())
```
Comment (VIKTORVAV99) [inline electricitymap/contrib/capacity_parsers/CA_ON.py:65]:
```suggestion
    fetch_production_capacity(ZoneKey("CA_ON"), datetime(2023, 3, 1), Session())
```

File: electricitymap/contrib/capacity_parsers/CA_ON.py [hunk 3]
```diff
@@ -0,0 +1,65 @@
...
+    df = df.rename(
+        columns={"Fuel Type": "mode", "Total Installed Capacity\n(MW)": "value"}
+    )
+    df = df[["mode", "value"]]
+    df["mode"] = df["mode"].apply(lambda x: MODE_MAPPING[x])
+    capacity = {}
+    for idx, data in df.iterrows():
+        capacity[data["mode"]] = {
+            "datetime": target_datetime.strftime("%Y-%m-%d"),
+            "value": round(data["value"], 2),
```
Comment (VIKTORVAV99) [inline electricitymap/contrib/capacity_parsers/CA_ON.py:55]: Why 2 in particular? Wouldn't 3 make more sense so we get kw resulution at least?

**Review by VIKTORVAV99 — APPROVED** [review]
Just some nitpicks but it seems to work as expected.

However the way that is documented in the readme don't really work? I had to run the file directly? Is this a future improvement or just missing for now?

File: electricitymap/contrib/capacity_parsers/CA_ON.py (hunk 3 above)
Comment (mathilde-daugy) [reply electricitymap/contrib/capacity_parsers/CA_ON.py:55]: no actually, i've been putting 0 for capacity, thanks for flagging!

**Pull request merged**